src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -704,6 +698,25 @@ public void scrollToPosition(By element) {
     PageObjectLogging.log("scrollToSelector", "Scroll to the web selector " + element.toString(), true);
   }
 
+  public void simulateScrollingToElement(By selector, int jumpSize, Duration duration) {
+    JavascriptActions jsActions = new JavascriptActions(driver);
+
+    final WebElement element = driver.findElement(selector);
+    final boolean scrollingToBottom = element.getLocation().getY() > driver.manage().window().getPosition().getY();
+    jumpSize = scrollingToBottom ? jumpSize : -jumpSize;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "jumpSize". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1226)